src/main/scala/com/databricks/spark/sql/perf/mllib/classification/RandomForestClassification.scala

```diff
@@ -0,0 +1,20 @@
+package com.databricks.spark.sql.perf.mllib.classification
+
+import org.apache.spark.ml.Estimator
+import org.apache.spark.ml.classification.RandomForestClassifier
+
+import com.databricks.spark.sql.perf.mllib._
+
+
+object RandomForestClassification extends TreeOrForestClassification {
+
+  override def getEstimator(ctx: MLBenchContext): Estimator[_] = {
+    import ctx.params._
+    // TODO: subsamplingRate, featureSubsetStrategy
+    // TODO: cacheNodeIds, checkpoint?
+    new RandomForestClassifier()
+      .setMaxDepth(depth.get)
```

Comment:
no need for the get if you use `import com.databricks.spark.sql.perf.mllib.OptionImplicits._`